app/src/views/DrefApplicationForm/Actions/NeedInput/i18n.json

```diff
@@ -1,6 +1,8 @@
 {
     "namespace": "drefApplicationForm",
     "strings": {
-        "drefApplicationRemoveNeed": "Remove Need"
+        "drefApplicationRemoveNeed": "Remove Need",
+        "drefApplicationLabel": "Current needs of population at risk ",
```

Comment:
```suggestion
        "drefApplicationLabel": "Current needs of population at risk",
```

Reply:
Lets also rename the key to more appropriate variable: maybe `currentNeedsLabel` or something similar